app/controllers/api/sessions_controller.rb

```diff
@@ -60,10 +60,7 @@ def refresh
       return
     end
 
-    session = Session.where(
-      'access_token = ? AND refresh_token = ?',
-      params[:access_token], params[:refresh_token]
-    ).first
+    session = Session.find_by_refresh_token(params[:refresh_token])
```

Comment:
Doesn't seem necessary to match on both access token and refresh token. Saved on adding a compound index as well by just querying the refresh token.

Reply:
It was required to match both tokens in order to prove the request is from an authentic user, i.e: clients should have this information stored, and should provide it when performing this request.

Reply:
I suppose it wouldn't hurt to check for equality after we find the correct session. Will make change.